src/rebar3_hank_prv.erl

```diff
@@ -3,6 +3,8 @@
 
 -export([init/1, do/1, format_error/1]).
 
+-define(FILES_PATTERN, "**/*.{erl,hrl,config,app.src,config.script,app.src.script}").
```

Comment:
Do we really need to read `config.script` files? I mean… they're escripts… how do we analyze them?

Reply:
Missed it, corrected!